### PR TITLE
documentation should explain the scheduletime  time zone info

### PR DESCRIPTION
**Pull request by fsi206914**
documentation should explain the scheduletime  time zone info

### CODE REVIEW

File: _includes/docs/latest/ajax-api.html [hunk 1]
```diff
@@ -1045,8 +1045,8 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
       <td>The name of the flow.</td>
     </tr>
     <tr>
-      <td>scheduleTime</td>
-      <td>The time to schedule the flow. Example: 12,00,pm,PDT</td>
+      <td>scheduleTime(with timezone)</td>
+      <td>The time to schedule the flow. Example: 12,00,pm,PDT (Unless UTC is specified, Azkaban will take current server's default timezone instead)</td>
```
Comment (HappyRay) [inline _includes/docs/latest/ajax-api.html]: Do you mean the specified time zone at the end will be ignored unless it is UTC?

**Comment by HappyRay** [comment]
Thanks for updating the doc!

File: _includes/docs/latest/ajax-api.html (hunk 1 above)
Comment (fsi206914) [inline _includes/docs/latest/ajax-api.html]: @HappyRay 
Yes. The code logic [here](https://github.com/azkaban/azkaban/blob/master/azkaban-webserver/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java#L694) might be not correct.

**Comment by logiclord** [comment]
LGTM

**Pull request merged**